Adding news to Wanderer space (#5932)
endless-sky/endless-sky: data/wanderer/wanderer news.txt

news "wanderer civilians, no dialog"
	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Gardener"
	message
		word
			"You see a Wanderer carefully place some seeds in the ground. They then spread some sort of light grey powder over them."
			"From the spaceport you can see some buildings with plants growing on their sides. A Wanderer prunes and waters them in mid-air."
			"While walking through the spaceport, you see a large greenhouse full of various plants. A Wanderer inside is tending to them."

> 			

Reviewer: I'm not sure that, frankly, any of these make a great deal of sense. 
Re:
1) These are their developed worlds, finished even. I doubt very much that the spaceport, the hub of their efforts, is in any way in need of seed planting in its vicinity.
2) Unless the Wanderers can hover with hummingbird like precision (which goes against their existing flight descriptions) _and_ they have hands separate from their wings (as yet unclear I think?) this is just straight up impossible.
3) Greenhouses take up a crapload of space, why would you locate this on ground with the highest demand for logistical considerations?

Author: I tried fixing them, are they better?

Reviewer: Honestly, not really. I would expect basic watering to be automated for an advanced species with a small population and a lot of ground to cover, and a farm has the same issue as a greenhouse for being next to the spaceport.
you could bend one of them like:
```suggestion
			"Many buildings near the spaceport have plants growing on their roofs. An indignant squawk draws your attention to a Wanderer as they rapidly flap away from an automated watering system dousing them where they stood."
```

Author: That's much better, thank you.

	name
		word
			"Group of Wanderers"
	message
		word

> 			"While walking through the spaceport, you see several Wanderers flying through the air. You duck to avoid them."

Reviewer: Are they flying or landing? Seems odd to duck if they're flying, especially since wanderers are implied to be rather taller than you are so nearby wanderers would have been even taller than you.

Author: I changed it, does it make more sense now?

Reviewer: It does make more sense yes. Feel free to resolve.

			"A crowd of Wanderers has formed around an object. Upon closer inspection, it appears to be some sort of technological device."
			"As you walk by a group of Wanderers, they begin to coo and chitter at you. Evidently they have not seen something like you before."
			
news "Wanderers and Korath"
	to show
		has "Wanderers: Sestor: Final: done"
	location
			government "Wanderer"
			not attributes "evacuation"
	name 
		word
			"Wanderer merchant captain"
			"Wanderer captain"
	message
		word
			"The Korath have [enormous, gigantic] ships that many of them live on their whole lives. I cannot [imagine, fathom] doing the same."
			"Have you [seen, viewed] a Korath up close? They only have feathers on their neck. How [strange, odd]."
			"The engines the Korath use have no regard for [effciency, subtlety]. But I must admit, they are much more powerful than ours."
			"The Korath cannot pay [well, adequately] for their transport jobs, but they insist on paying [despite it, nonetheless].

> 			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of [those we work with, our peers]."

Reviewer: This final translation issue feels off. Maybe more like:
```suggestion
			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of those who we are [undertaking, rescuing]."
```

	name
		word
			"Wanderer linguist"
			"Linguist"
	message
		word

> 			"Many scientists are currently [working on, engineering] a translation device that will allow direct [communication, dialog] with the Korath in their own language. It is [exciting, satisfying] to be part of this effort.

Reviewer: The wanderers took the time to learn Hai rather than do such a thing. This seems an incongruous entry.

Author: I would argue that developing a translation device for the Korath is a more urgent task than developing one for the Hai, because the Wanderers are going to be directly working with Korath for a long time, probably millennia. It's probably easier for them to learn Hai than build a translation device that will probably not be needed once the transition through the Eye is complete.

Reviewer: Perhaps, but someone has to learn it before they can do that effectively anyway. Perhaps, given the early stage of things, the present focus should be on whether or not it is possible for a Wanderer to fully speak the Korath language, and be assessing the need or scope requirements for a translator? Just like, dial it back a step?

Author: Alright, that makes sense.

			
news "Wanderers and Aliens"
	to show
		has "Wanderers Conversation: offered"
	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Wanderer merchant captain"
			"Wanderer captain"
			"Wanderer civilian"
			"Wanderer"
	message
		word
			"Long ago, we [rebirthed, cleansed] a group of worlds in the south. I wonder if any [aliens, sentient beings] live there now?"

> 			"It is a shame we must [fight, defend against] the Hai. I wish we had the opportunity to [meet, socialize with] their peaceful brethren."

Reviewer: Alternative
```suggestion
			"It is a shame we must [fight, defend against] the Hai. I wish we had the opportunity to [engage, treat] with their peaceful brethren."
```

Author: I don't think the phrase `treat with their peaceful brethren` makes sense.

Reviewer: So for reference this is the phrase usage: https://www.macmillandictionary.com/dictionary/british/treat-with#:~:text=(treat%20with%20someone)%20to%20have,reach%20an%20agreement%20about%20something

The wanderers are often oddly formal, so I figured that this was an appropriate usage for two peoples who haven't met managing to do so.

			"Alien societies [fascinate, interest] me. I [hope, desire] for more opportunities to meet with them."
	name
		word
			"Wanderer linguist"
			"Linguist"
	message
		word

> 			"The Hai language is [complicated, convoluted], but learning it has yielded much [insight, knoweldge] about the Hai and [aliens, non-Wanderers] in general."

Reviewer: Typo + the wanderers are meant to be well old. I doubt that the Hai language provides much insight to anyone but the Hai that they couldn't have thought about from meeting other aliens in the past.
```suggestion
			"The Hai language is [complicated, convoluted], but learning it has yielded much [insight, knowledge] about the Hai in general."
```

			"It is difficult to [pronounce, speak] Hai words, because their [physiology, structure] is so much different than ours." 
			
news "Wanderer civilians, dialog"
	to show
		has "Wanderers Conversation: offered"
	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Wanderer"
			"Wanderer worker"
			"Wanderer traveller"
	message
		word
			"Whenever I go on a [journey, trip] through space I always bring some of my [favorite, most cherished] potted plants. It gets [lonely, barren] when you are far from nature." 
			"I have [heard of, known of] a human captain in Wanderer space, but seeing you is still quite [strange, odd]."
			"I cannot [imagine, fathom] being unable to fly. How do you manage to get anywhere?"
			"I wonder what kinds of [creatures, beings] live on your home planet."

> 			"With that jump drive, you must be able to [see, witness] all kinds of wonderous [creatures, forms of life] from across the galaxy. I wish I had such access."

Reviewer: Alternative simplification:
```suggestion
			"With that jump drive, you must be able to [observe, witness] all kinds of wondrous life forms from across the galaxy. I wish I had such access."
```

news "Wanderer scientists"
	to show
		has "Wanderers Conversation: offered"
	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Wanderer scientist"
			"Scientist"
			"Wanderer ecologist"
			"Ecologist"
	message
		word
			"I recently [travelled, journeyed] to another world on a scientific [mission, study], where we discovered a new species of plant. How [exciting, thrilling]!"

> 			"The best part of my [job, task] is the thrill of [discovery, finding new knoweldge]."

Reviewer: Alternative:
```suggestion
			"The best part of my [job, task] is the thrill of [discovery, revelation]."
```

news "Wanderer soldier"
	to show
	has "Wanderers Conversation: offered"
	location
		government "Wanderer"
		attributes "evacuation"
	name
		word
			"Wanderer soldier"
			"Soldier"
			"Wanderer conscript"
			"Conscript"
	message
		word
			"Society does not [like, appreciate] what I do, but it is [necessary, essential] in times like these."
			"I wish I did not have to [fight, do battle] for a living. It is a [savage, unelegant] way to get by."
			"I joined the [defense fleet, navy] because I wanted to protect lifeforms. Instead I am [battling, killing] them. It brings me great [sadness, sorrow] to see such [destruction, waste]."
			"I should've [become, studied to be] an ecologist like my parents [wanted, desired] me to."

> 			"I kill one so that [others, several] may be [saved, spared]."

Reviewer: I'm not honestly sure that any of this is in keeping with the Wanderer's as they've been described thus far.

Author: Could you elaborate?

Reviewer: The conversations in the plot imply that the wanderers simply did not engage in combat until extremely recently. The idea that any of them are now soldiers without having had a different career prior to that is just incongruous.
Also if this only fires on the evacuation planets it's a very narrow window where they're able to show up.

Author: On Varu K'est, the description after the Unfettered invade implies that the Wanderers did have a standing army before the invasion: `Before the Unfettered invaded, the only Wanderer settlements remaining here were the military bases they had set up to defend their frontier.` I could change this to have it only spawn on that world.

Reviewer 2: How feasible is it to have this only appear on that world until after the Unfettered take the first Wanderer worlds?

Author: I'm very new to ES syntax, but perhaps I could use a combo of `government: Wanderer` and `attributes: wanderer military`?

Author: @MasterOfGrey are you still opposed to these lines? I just want your opinion before I do anything with them.

Reviewer: On Varu K'est it has the description you mentioned, but very early in Wanderers Iktat Rek says directly to your face, `"At first only a single Hai ship [visited, invaded] us, but back then we had no defenses in place to stop it."`

Implying that they didn't actually have those bases before the unfettered invaded, as the description suggests.

So It comes down to whether you're in favour of the narrative canon or the written world base. 

I tend to lean on the narrative canon: which would suggest that none of these lines should be used, and the world description also needs to change.

On the other hand, you could justify altering the narrative canon to fit the world base instead, in which case these are fine, but I feel like that could introduce more issues with why it took the Wanderer's so long to dig up and get back into rebuilding their military ships.

Author: I think you're right, the narrative makes more sense than the world description in terms of the Wanderers campaign. That should probably get changed.
I could change these lines to only show up after the Unfettered invasion, and edit them to imply that their military was only very recently put together.

Reviewer: That would definitely be workable, and flavourful.

Author: I tried editing some lines and adding others, is it better now?

Reviewer: I would lose the conscripted one, but otherwise I'd say these are excellent.

Author: The `"I was [conscripted, forced] into service in a warship...` one?

Reviewer: Yeah that one, the wanderers seem perfectly willing to pull together in a crisis. Conscription shouldn't be necessary.

			"The Korath cannot pay [well, adequately] for their transport jobs, but they insist on paying [despite it, nonetheless]."
			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of those who we are [undertaking, rescuing]."
			"Have you been on a Korath [world, planet]? It is a [sobering, depressing] reminder of the importance of our [task, work]."
			"I am [excited, overjoyed] to be part of [healing, renewing] the Korath worlds. We are once again [transforming, altering] the old into the new."
		word
			`"`
	name
		word
			"Wanderer linguist"
			"Linguist"
	message
		word
			`"`
		word
			"Many scientists are currently [studying, examining] the Korath language, and attempting to teach it to others. It is [exciting, satisfying] to be part of this [effort, undertaking]."
			"We are still [figuring out, determining] whether or not a Wanderer who has not undergone the Molt could [learn, apprehend] the Korath language. The sounds in their [speech, tongue] are hard for us to [speak, pronounce]."
		word
			`"`
			
news "Wanderers and Aliens"
	to show

> 		has "Wanderers Conversation: offered"

Reviewer 3: It might make more sense to use the following condition. This could be done for the other news entries below this.
```suggestion
		has "language: Wanderers"
```

	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Wanderer merchant captain"
			"Wanderer captain"
			"Wanderer civilian"
			"Wanderer"
	message
		word
			`"`
		word
			"Long ago, we [rebirthed, cleansed] a group of worlds in the south. I wonder if any [aliens, sentient beings] live there now?"
			"It is a shame we must [fight, defend against] the Hai. I wish we had the opportunity to [engage, treat] with their peaceful brethren."
			"Alien societies [fascinate, interest] me. I [hope, desire] for more opportunities to meet with them."
			"I [wonder, ponder] how a human managed to get all the way here. Aliens in our [space, territory] are a [rare, precious] sight."
			"It is not often our [species, culture] gets to meet with so many [different, varieties of] aliens at once."
			"I have heard that Hai and human space are [linked, joined] with a wormhole. Could it have been made by [our caretakers, those that open the Eye]?"
			"The Unfettered Hai enter Wanderer [space, territory] using jump drives. Where did they get such [devices, technology]?"

> 			"Our species has [met, made contact with] with many aliens before, but never so many at once."

Reviewer 3: Double with.
```suggestion
			"Our species has [met, made contact] with many aliens before, but never so many at once."
```

			"With all the different species we have to [keep track of, document], it is easy to get [mixed up, confused]. A spiny-haired T'kai may be upon closer [inspection, study] a short-necked Kat'tchi."
		word
			`"`

news "Wanderer soldier"
	to show
		has "Wanderers Invaded 0: accepted"
	location
		government "Wanderer"
	name
		word
			"Wanderer soldier"
			"Soldier"
			"Wanderer conscript"
			"Conscript"
	message
		word
			`"`
		word
			"I wish we did not have to [fight, do battle] to survive. It is a [savage, inelegant] way of solving conflict."
			"I joined our new [defense fleet, navy] because I wanted to protect lifeforms. Instead I am [battling, killing] them. It brings me great [sadness, sorrow] to see such [destruction, waste]."
			"I volunteered for service in a warship, and many [ask, ponder] why. For every Hai I [shoot down, kill] I save [many, multiple] civilians."
			"Just a few days ago I was asked to form a new [military, army] unit, even though I was given almost no [training, experience] in war. These are [desperate, hopeless] times indeed."

> 			"Our [commander, unit leader] used to be a zoologist. His only [combat, fighting] experience was defending himself against a rabid Chi'kta on an [expedition, study]. They couldn't find anyone [better, more suitable], so he got 'promoted'."

Reviewer 3: ```suggestion
			"Our [commander, unit leader] used to be a zoologist. His only [combat, fighting] experience was defending himself against a rabid Chi'kta on an [expedition, study]. They couldn't find anyone [better, more suitable], so he got 'promoted.'"
```

		word
			`"`

news "Wanderer engineer"
	to show
		has "Wanderers Solifuge Recon 2: offered"
	location
		government "Wanderer"
		not attributes "evacuation"
	name
		word
			"Wanderer engineer"
			"Engineer"
	message
		word
			`"`
		word
			"The warships we have begun to [manufacture, build] are from a more [brutish, dark] time in our history. I hope that we are not headed towards such times [again, once more]."
			"Despite their [savage, regrettable] purpose, the warships we have begun to [make, fashion] are impressive from a purely technological [perspective, view]."
			"I regret that I must [recreate, unearth] technologies of [war, battle], but it is necessary for our [survival, existence]."
			"I would like to [study, understand] human technology and its [capabilities, abilities], but I am not able to. Now that we must build [warships, battleships], all my time has been taken up by their [production, creation]."
			"I create weapons of [destruction, annihilation] so that we may survive and go on to [create, rebirth]."
		word
			`"`

news "Wanderers and the Eye"
	to show

> 		has "Wanderers Ap'arak 3: completed"

Reviewer 3: Same as before, a more appropriate condition might be the following:
```suggestion
		has "event: wanderers: the eye opens"
```
(Both the suggestions are to use synonymous conditions in terms of when they appear, I'd just prefer using the condition that is the most direct trigger for these messages displaying instead of using the mission that triggers the condition/event that these are based on.)

	message
		word
			`"`
		word
			"The Eye has opened in my [lifetime, existence]! I never dreamed of such a [miracle, wonder] happening for me to see!"
			"The lands beyond the Eye shall bring a [rebirth? renewal?] to our species."
			"It is time for our [society, species] to move to new lands; the Eye have given us this [honor, opportunity]."
			"The opening of the Eye is a sign that we must continue our [purpose? existence?] in a new [part, region] of this galaxy."
			"What [wonders, changes] await us at the end of the Eye!"
			"I am honored to [bear witness? attest?] to this new phase of our cycle. The Eye shall [renew?, change?] us."
			"Now that the Eye is [here, upon us], I wonder if I'll [undergo, experience] the Molt. It would be a [honor, privilege] to be changed in this way."
			"Some of my feathers have [begun, started] to fall out, and I've been [noticing, observing] lumps on my skin. I [hope, wish] that it's the Molt."
		word
			`"`

news "Wanderer oil drilling"
	to show
		has "Wanderers Conversation: offered"
	location
		government "Wanderer"
		attributes "oil"
	name
		word
			"Safety inspector"
			"Wanderer safety inspector"
	message
		word
			`"`
		word

> 			"My job is to prevent the [destruction, death] of that which we have sought to [renew, nurture.]"

Reviewer 3: ```suggestion
			"My job is to prevent the [destruction, death] of that which we have sought to [renew, nurture]."
```
